src/content_sao.cpp

```diff
@@ -617,6 +617,9 @@ void LuaEntitySAO::notifyObjectPropertiesModified()
 
 void LuaEntitySAO::setVelocity(v3f velocity)
 {
+	if(util_isnan(velocity.X) || util_isnan(velocity.Y) ||
+			util_isnan(velocity.Z))
```

Comment:
Just a suggestion, maybe it would be more neatly formatted to write this if statement as:
```
	if (util_isnan(velocity.X) ||
		util_isnan(velocity.Y) ||
		util_isnan(velocity.Z)) {
		...
	}
```